standard/sections/clause_3_references.adoc

```diff
@@ -2,7 +2,7 @@
 
 * [[ogcapi-features]] OGC: OGC 17-069r, OGC API - Features - Part 1: Core 1.0.1 (2022) footnote:[https://docs.opengeospatial.org/is/17-069r4/17-069r4.html]
 * [[rfc7946]] IETF: RFC-7946 The GeoJSON Format (2016) footnote:[https://datatracker.ietf.org/doc/html/rfc7946]
-* [[rfc8259]] IETF: RFC-8259 The JavaScript Object Notation (JSON) Data Interchange Formathe GeoJSON Format (2016) footnote:[https://datatracker.ietf.org/doc/html/rfc8259]
```

Comment:
fixed typo

Reply:
Now fixed in https://github.com/wmo-im/wis2-notification-message/commit/edb6593d3cfd268ddfefc854d607594126d0843f